libs/api/domains/air-discount-scheme/src/lib/api-domains-air-discount-scheme.resolver.ts

```diff
@@ -0,0 +1,62 @@
+import { UseGuards } from '@nestjs/common'
+import { Query } from '@nestjs/graphql'
+import type { User } from '@island.is/auth-nest-tools'
+import { ApiScope } from '@island.is/auth/scopes'
+import {
+  CurrentUser,
+  IdsAuthGuard,
+  IdsUserGuard,
+  Scopes,
+  ScopesGuard,
+} from '@island.is/auth-nest-tools'
+import { Audit } from '@island.is/nest/audit'
+import { AirDiscountSchemeService } from './api-domains-air-discount-scheme.service'
+
+import {
+  Discount as TDiscount,
+  User as TUser,
+} from '@island.is/air-discount-scheme/types'
+import { Discount } from '../models/discount.model'
+
+type DiscountWithTUser = Discount & { user: TUser }
+
+const TWO_HOURS = 7200
+
+@UseGuards(IdsAuthGuard, IdsUserGuard, ScopesGuard)
+@Scopes(ApiScope.ads)
+//@Audit({ namespace: '@island.is/air-discount-scheme' })
+export class AirDiscountSchemeResolver {
+  constructor(private airDiscountSchemeService: AirDiscountSchemeService) {}
+
+  @Query(() => [Discount], { nullable: true })
+  @Audit()
+  async adsGetDiscount(@CurrentUser() user: User) {
```

Comment:
The resolver name is `AirDiscountSchemeResolver`, unnecessary to prefix methods with `ads`.
```suggestion
  async getDiscount(@CurrentUser() user: User) {
```